test/integration/core-js.js

```diff
@@ -1,2 +0,0 @@
-
-require("core-js").Array.prototype.map.call([], () => {});
```

Comment:
Normally we shouldn't be removing tests. 

But it seems like test coverage remains high so this is probably fine.